x-pack/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -292,9 +305,13 @@ export class HeadlessChromiumDriver {
       }
 
       if (!allowed || !this.allowRequest(interceptedUrl)) {
-        logger.error(`Got disallowed URL "${interceptedUrl}", closing browser.`);
         this.page.browser().close();
-        throw new Error(`Received disallowed URL in response: ${interceptedUrl}`);
+        logger.error(
+          i18n.translate('xpack.reporting.chromiumDriver.disallowedOutgoingUrl', {
+            defaultMessage: `Received disallowed URL in response: "{interceptedUrl}". Closing the browser.`,
+            values: { interceptedUrl },
+          })
+        );
```

Comment:
These `throw new Error` were happening inside of listeners, and causing the server to crash.